pod/perlsub.pod

```diff
@@ -52,8 +52,8 @@ X<subroutine, call> X<call>
 
 Like many languages, Perl provides for user-defined subroutines.
 These may be located anywhere in the main program, loaded in from
-other files via the C<do>, C<require>, or C<use> keywords, or
-generated on the fly using C<eval> or anonymous subroutines.
+other files via the L<C<do>|perlfunc/do>, L<C<require>|perlfunc/require>, or L<C<use>|perlfunc/use> keywords, or
```

Comment:
There is no "do" or "use" entry in perlfunc; the section links should be "do EXPR" and "use Module"

Reply:
OK will correct, but why the preview links "correctly" to https://metacpan.org/pod/perlfunc#do (or `-EXPR`) ?

Reply:
Metacpan and perldoc sites have an override that generates additional link anchors to make CPAN crosslinking easier, but core docs should still use the correct section links internally.